package.json

```diff
@@ -4,12 +4,19 @@
   "description": "Benchmark project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Consider adding a testing command to make testing easier.  (P.S. Where are the tests??)